content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -88,7 +80,7 @@ restricted to the cluster administrator. This is typically the default. See [Aut
 Overview](/docs/reference/access-authn-authz/authorization/) for more details.
 {{< /note >}}
 
-### Usage
+## Usage
```

Comment:
Line 115, 127: Check the spacing around the tooltip (heading). Should a tooltip appear in a heading?
```
#### {{< glossary_tooltip term_id="containerd" >}}
#### {{< glossary_tooltip term_id="cri-o" >}}
```